examples/cms-sanity/pages/index.js

```diff
@@ -36,7 +37,7 @@ export default function Index({ allPosts, preview }) {
 }
 
 export async function getStaticProps({ preview = false }) {
-  const allPosts = await getAllPostsForHome(preview)
+  const allPosts = overlayDrafts(await getClient(preview).fetch(indexQuery))
```

Comment:
Maybe it's just the naming of this (`overlayDrafts`) but I'm pretty confused as to what this does.